File: Assignment47&50/program19.java
//write java program which accept N numbers from user and accept range,Display all elements from that range
import java.lang.*;
import java.util.*;



class NumberX
{
    public void Display(int Arr[],int iStart, int iEnd)
    {
        int i = 0;
        System.out.println("Number Are : ");
        for(i = 0; i<Arr.length; i++)
        {
            if((Arr[i] >= iStart) && (Arr[i] <= iEnd))
            {
                System.out.println(Arr[i]);
            }
        }
    }
}


class program19
{
    public static void main(String arg[])
    {
        
   
        
        int iValue = 0,i = 0,iStart = 0,iEnd = 0;
        int iRet = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Number");
        iValue = sobj.nextInt();

        System.out.println("Enter the  Start Number");
        iStart = sobj.nextInt();

        System.out.println("Enter the End Number");
        iEnd = sobj.nextInt();


        int Arr[] = new int[iValue];
        System.out.println("Enter the Elements");
        for(i = 0; i< Arr.length;i++)
        {
            Arr[i] = sobj.nextInt();
        }

        NumberX obj = new NumberX();
        obj.Display(Arr,iStart,iEnd);
       
    }
}

File: Assinment3&4/program19.c
#include<stdio.h>

void DisplayEvenFActor(int iNo)
{
    int i = 0;
    if(iNo<=0)
    {
        iNo = -iNo;
    }
    for(i = 1;i<=iNo;i++)
    {
        if((iNo % i)==0)
        {
            printf("%d\n",i);

        }
    }
}
int main()
{
    int iValue = 0;

    printf("Enter number\n");
    scanf("%d",&iValue);
    DisplayEvenFActor(iValue);

    return 0;
}